src/drawing/DrawingAnnotations.tsx

```diff
@@ -119,7 +121,12 @@ const DrawingAnnotations = (props: Props): JSX.Element => {
             )}
 
             {isCreating && (
-                <DrawingCreator className="ba-DrawingAnnotations-creator" onStart={handleStart} onStop={handleStop} />
+                <DrawingCreator
+                    className="ba-DrawingAnnotations-creator"
+                    color={color}
```

Comment:
Could also use the `useSelector` hook inside DrawingCreator to avoid having to pass this down.

Reply:
Stroke has two properties, `color` and `size`. Using `useSelector` for color and default value for size seems not consistent. Maybe we can refactor it if we add `size` to the store in the future.